### PR TITLE
fix(acl): command scope should not error out if missing

### PR DESCRIPTION
**Pull request by lucasfernog**
The command-specific scope is not required and it should be easier for plugin devs to reference them without having to deal with that:
```rust
#[tauri::command]
async fn execute(scope: tauri::command::CommandScope<'_, ScopeEntry>)...
```
so it should not fail to retrieve the value if the scope was not set by the app user

### CODE REVIEW

**Comment by lucasfernog** [comment]
Usage example: [tauri-apps/plugins-workspace@`1f307c7` (#862)](https://github.com/tauri-apps/plugins-workspace/pull/862/commits/1f307c788b6c20824064494ccb75ca177122ba12)

File: core/tauri/src/command/authority.rs [hunk 1]
```diff
@@ -112,9 +112,27 @@ impl<T: Debug + DeserializeOwned + Send + Sync + 'static> ScopeValue<T> {
   }
 }
 
+#[derive(Debug)]
+enum OwnedOrRef<'a, T: Debug> {
```
Comment (amrbashir) [inline core/tauri/src/command/authority.rs:116]: Can't we use `Cow` ?

**Review by amrbashir — DISMISSED** [review]

File: core/tauri/src/command/authority.rs (hunk 1 above)
Comment (lucasfernog) [reply core/tauri/src/command/authority.rs:116]: Cow requires `T: Clone` and we don't really need that :(

Comment (amrbashir) [reply core/tauri/src/command/authority.rs:116]: I see, it is fine then

**Review by amrbashir — APPROVED** [review]

**Pull request merged**